consensus/quorum/one-node-staked-vote.go

```diff
@@ -0,0 +1,104 @@
+package quorum
+
+import (
+	"math/big"
+
+	"github.com/harmony-one/bls/ffi/go/bls"
+	"github.com/harmony-one/harmony/internal/common"
+	"github.com/harmony-one/harmony/numeric"
+	"github.com/harmony-one/harmony/staking/effective"
+)
+
+var (
+	twoThirds = numeric.NewDec(2).QuoInt64(3)
+)
+
+type stakedVoter struct {
+	isActive, isHarmonyNode bool
+	effective               numeric.Dec
```

Comment:
I bet this is the amount of stake? Why it's Dec? It should be Int as it's the number of token, right? (Please add comment)

Reply:
Preference of using the Dec (wrapper around big.Int) is ease of use of it, good methods it provides.